specification/compute/resource-manager/readme.md

```diff
@@ -228,6 +228,12 @@ directive:
       - NestedResourcesMustHaveListOperation
     reason:
       - CRP supports the list /restorePoint operation by allowing customers to call Get RestorePointCollection with $expand=RestorePoints
+  - where:
+      - $.definitions.ScaleUnit
+    suppress:
+      - NestedResourcesMustHaveListOperation
+    reason:
+      - CRP cannot support the list API for scale units due to the undesired load that would inflict on the system
```

Comment:
You should put these suppressions in the readme.md of your own RP instead of this file.
Please revert this file and change `specification/azsadmin/resource-manager/compute/readme.md` instead

Reply:
Fixed! Thanks.